src/table.jl

```diff
@@ -12,13 +12,22 @@ struct Table{T}
         new{T}(shp, dbf)
     end
 end
-
 function Table(shp::Handle{T}, dbf::DBFTables.Table) where {T}
     Table{T}(shp, dbf)
 end
 
+"""
+Creates a struct array of shp and dbf 
+"""
+function structarray(shp::Handle, dbf::DBFTables.Table)
+    dbf_cols = Tables.columntable(dbf)
+    meta = collect(GB.meta(s) for s in shp.shapes)
+    meta_cols = Tables.columntable(meta)
+    return StructArray(Geometry = collect(GB.metafree(i) for  i in shp.shapes); meta_cols..., dbf_cols...)
```

Comment:
So now that we have implemented a new design in GeoJSONTables visr/GeoJSONTables.jl#3 for metadata. I am wondering if the method we are using here is valid?
I mean, here we construct the metageometry, and have to break it down at the end for making a StructArray.
Do we keep using this approach? I am not sure if breaking down the meta geometry will have performance effects? Or we can go the GeoJSONTables way and keep meta and geometries separate from the beginning?